src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
```

Comment:
do not reset margins with the `*` selector. do it only for the tags that you need to reset margins for